gui/src/lib/api.ts

```diff
@@ -194,6 +194,7 @@ export interface DescribeTasksInput {
 
 export interface DirectoryEntry {
   name: string;
+  path: string;
```

Comment:
@BenElgar was I right or wrong to add this here? Typescript was bugging me about it and it seems like it _is_ included in the results.

Reply:
Yes you were, right, sorry. I added it to the back end as well: [`bf9bff7` (#343)](https://github.com/deref/exo/pull/343/commits/bf9bff7261dedcf854fd36f3ec416f6aafbc69b9)